UUIDs.txt

```diff
@@ -1129,7 +1130,7 @@
 0xFE84 RF Digital Corp
 0xFE85 RF Digital Corp
 0xFE86 Huawei Technologies Co., Ltd.
-0xFE87
+0xFE87 Unknown
```

Comment:
I actaully found the name for that value: 
`Qingdao Yeelink Information Technology Co., Ltd. `

so maybe just add that and then merge

Reply:
Okay will do so!